band_together/scenes/scene_transition/scene_transition.gdshader

```diff
@@ -8,11 +8,6 @@ uniform float progress : hint_range(0, 1);
 uniform float diamondPixelSize = 10.0;
 
 void fragment() {
-	float xFraction = fract(FRAGCOORD.x / diamondPixelSize);
-	float yFraction = fract(FRAGCOORD.y / diamondPixelSize);
-	float xDistance = abs(xFraction - 0.5);
-	float yDistance = abs(yFraction - 0.5);
-	if (xDistance + yDistance + UV.x + UV.y > progress * 4.0) {
-		discard;
-	}
+	vec2 dist = abs(fract(FRAGCOORD.xy/diamondPixelSize)-0.5);
+    COLOR.a = step(dist.x+dist.y+UV.x+UV.y-progress*4.0,0.0);
```

Comment:
Not sure why it's showing the indent weird, but it looks normal when I pull it up in my game engine.